src/routes/(app)/checkout/+page.server.ts

```diff
@@ -185,8 +185,8 @@ export const actions = {
 			? z
 					.object({
 						billing: z.object({
-							firstName: z.string().min(1),
-							lastName: z.string().min(1),
+							firstName: z.string().optional().default(''),
+							lastName: z.string().optional().default(''),
```

Comment:
you should check `isCompany` first, and then change this rule based on `isCompany`.

And you only need one of `optional()` and `default()`, not both